lesson_05/homework_05.py

```diff
@@ -1,39 +1,115 @@
 small_list = [3, 1, 4, 5, 2, 5, 3]
 big_list = [3, 5, -2, -1, -3, 0, 1, 4, 5, 2]
+base_dict = {'contry':'Ukraine', 'continent': 'Europe', 'size': 123}
+add_dict = {"a":1, "b":2, "c":2, "d":3, 'size': 12}
+
+
 # task 1. Знайдіть всі унікальні елементи в списку small_list
+small_list_unique = [element for element in small_list if small_list.count(element) == 1]
+print(f"Унікальні елементи в списку {small_list} -> {small_list_unique}")
+
 
 # task 2. Знайдіть середнє арифметичне всіх елементів у списку small_list
+arithmetic_mean = sum(small_list) / len(small_list)
+print(round(arithmetic_mean, 4))
```

Comment:
```suggestion
print(int(arithmetic_mean))
```
нам тут прям суперточність не треб

Reply:
Зрозумів.